cache/.htaccess

```diff
@@ -1 +0,0 @@
-Deny from all
```

Comment:
This file should be removed from the PR. It came up for me to today, looks like something new.